packages/manager/apps/nasha-react/src/components/layout-helpers/list/dataGrid.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, Link } from 'react-router-dom';
+import './dataGrid.scss';
+
+const Datagrid = (props: { data }) => {
+  const { data } = props;
+  const navigate = useNavigate();
+  const tableHeaders = Object.keys(data[0]);
+  const { t } = useTranslation('nasha-react');
+  return (
+    <table>
+      <thead>
+        <tr>
+          {tableHeaders.map((header) => (
+            <th key={header}>{t(header)}</th>
```

Comment:
```suggestion
            <th>{t(header)}</th>
```

Is the key really usefull ?